CODEOWNERS

```
@@ -1,7 +1,12 @@
 # Global owners
 * @cloudflare/wrangler
+/.changeset/ @cloudflare/wrangler @cloudflare/pages @cloudflare/d1
+/fixtures/ @cloudflare/wrangler @cloudflare/pages @cloudflare/d1
+*.test.* @cloudflare/wrangler @cloudflare/pages @cloudflare/d1
```

what about this instead?

```suggestion
/packages/wrangler/src/__tests__/pages @cloudflare/pages 
/packages/wrangler/src/__tests__/d1 @cloudflare/d1
```

i think this would achieve the same desired result while reducing the noise on requested reviews and also keeping the wrangler team as codeowners on wrangler core tests

---

I _was_ actually trying to give D1 and Pages permission to edit Wrangler-core tests. My thinking was that we couldn't be affecting anything properly functional (since the actual code was still CODEOWNER'd by the core team), but that in case we somehow were affecting core tests with our changes (e.g. changing the content of the Pages beta warning label, or the `--help` description of a command we owned), we wouldn't need to seek core approval.

---

How often is that likely that such test files will be affected?

I would argue that in these cases it is not a hardship to get core approval, especially if as suggested the changes will be minimal/trivial.

The counter case is that a major change to a core test is applied without the core team noticing.